extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeFooter.php

```diff
@@ -4,13 +4,17 @@
 class NodeFooter extends Node {
 
 	public function getData() {
-		return [ 'value' => $this->getExternalParser()->parseRecursive(
-			\Wikia\PortableInfobox\Helpers\SimpleXmlUtil::getInstance()->getInnerXML( $this->xmlNode )
-		) ];
+		if ( !isset( $this->data ) ) {
+			$this->data = [ 'value' => $this->getInnerValue( $this->xmlNode ) ];
+		}
+
+		return $this->data;
 	}
 
-	public function isEmpty( $data ) {
-		$links = trim( $data['value'] );
+	public function isEmpty() {
+		$data = $this->getData();
+		$links = trim( $data[ 'value' ] );
+
 		return empty( $links ) && $links != '0';
```

Comment:
lets leave it for now, we should go through isEmpty logic one more time later in time